fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1273,6 +1273,11 @@ public void abortTimeoutTxns(long currentMillis) {
     public void replayUpsertTransactionState(TransactionState transactionState) {
         writeLock();
         try {
+            if (transactionState.getTransactionStatus() == TransactionStatus.UNKNOWN) {
+                LOG.info("remove unknown transaction: {}", transactionState);
+                deleteTransaction(transactionState);
```

Comment:
this function deleteTransaction() only delete the final state txn, maybe you should write another function to delete the txn from the running map

Reply:
Why does it appear in the running map?